cogs/quote_commands.py

```diff
@@ -112,6 +100,24 @@ async def quote(self, ctx:utils.Context, messages:commands.Greedy[discord.Messag
                 embed.description = text
             # embed.set_footer(text=f"Quote ID {quote_id.upper()}")
             embed.timestamp = timestamp
+            return embed
+
+
+        
+        
```

Comment:
What's with all this whitespace my guy? You want one newline seperating functions in classes.